### PR TITLE
Thrusters/Weapons/Turret Technical Information

### PR DESCRIPTION
**Pull request by Ashparty**
Added **USEFUL** Technical information to Starship Weapons, Turrets and Thrusters

### CODE REVIEW

File: Starships/Weapons.md [hunk 1]
```diff
@@ -2,30 +2,82 @@
...
 
+> Explosion Power: 0
+
+> Power Usage: 500
+
+> Range: 120
+
+> Speed: 150
+
+> Max-Per-Shot: N/A
```
Comment (Peter-Crawley) [inline Starships/Weapons.md:47]: If a value is N/A just don't include it.

File: Starships/Weapons.md [hunk 2]
```diff
@@ -2,30 +2,82 @@
...
 
+> Explosion Power: 4
+
+> Power Usage: 2500
+
+> Range: 160
+
+> Speed: 400
+
+> Max-Per-Shot: 2
```
Comment (Peter-Crawley) [inline Starships/Weapons.md:17]: Don't need the dashes

File: Starships/Weapons.md [hunk 3]
```diff
@@ -2,30 +2,82 @@
...
 
 {INSERT IMAGE HERE}
 
+> Explosion Power: 2
+
+> Power Usage: 50000
+
+> Range: 140
+
+> Speed: (500*1000) per second (May be innaccurate)
```
Comment (Peter-Crawley) [inline Starships/Weapons.md:77]: This one is weird.

Phasers have an initial speed which after a certain amount of time gets switched to be really really fast.

File: Machines/Turrets.md [hunk 4]
```diff
@@ -23,6 +23,15 @@ Materials:
...
 > 1x Oak Sign
 
+Technical Information:
+> Power Usage: 6000
+
+> Explosion Power: 4
+
+> Projectiles: 1
+
+> Cooldown: 0.25
```
Comment (Peter-Crawley) [inline Machines/Turrets.md:33]: Cooldown really should be done in terms of seconds or milliseconds

**Comment by Peter-Crawley** [comment]
I can't select request changes on the review. :/

GitHub's mobile GUI sucks.

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

File: Machines/Turrets.md (hunk 4 above)
Comment (Ashparty) [reply Machines/Turrets.md]: Done

File: Starships/Weapons.md (hunk 2 above)
Comment (Ashparty) [reply Starships/Weapons.md]: Removed Dashes

File: Machines/Turrets.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ Technical Information:
 
 > Projectiles: 1
 
-> Cooldown: 0.25
+> Cooldown: 25ms
```
Comment (Peter-Crawley) [inline Machines/Turrets.md:33]: These don't seem correct.

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

File: Machines/Turrets.md (hunk 5 above)
Comment (Peter-Crawley) [reply Machines/Turrets.md:33]: There are 1000 milliseconds in 1 second, so I think you meant 250ms.

**Pull request merged**